client/blocks/checklist/checklist-task.jsx

```diff
@@ -86,8 +86,11 @@ export class ChecklistTask extends PureComponent {
 						className="checklist__task-action"
 						onClick={ this.handleClick }
 						primary={ buttonPrimary }
+						title={ buttonText }
 					>
-						{ hasActionlink ? completedButtonText : buttonText }
+						{ hasActionlink
+							? completedButtonText
+							: translate( '%(buttonText)s!', { args: { buttonText: buttonText } } ) }
```

Comment:
If we apply a translation here, a translator will see `[ someVariable ]!`. The can only see an exclamation point to translate, the rest of the text is hidden from them 😮 🙈 

It's a good idea to apply the translation to the whole string, so we should drop the changes on this line and apply the translations elsewhere 👍